src/hooks/useUniversalRouter.ts

```diff
@@ -3,14 +3,14 @@ import { BigNumber } from '@ethersproject/bignumber'
 import { t } from '@lingui/macro'
 import { sendAnalyticsEvent } from '@uniswap/analytics'
 import { SwapEventName } from '@uniswap/analytics-events'
+import { sendTransaction } from '@uniswap/conedison/provider/index'
```

Comment:
should this be configured at the webpack level to resolve as an alias?

Reply:
I couldn't figure it out and didn't want to sink too much time into it.

Reply:
i meant inside the webpack config for interface, same way we've alias'd the import for `conedison` to the dist